src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1749,27 +1749,35 @@ private DiagnosticInfo GetBadEventUsageDiagnosticInfo(EventSymbol eventSymbol)
                 new CSDiagnosticInfo(ErrorCode.ERR_BadEventUsageNoField, leastOverridden);
         }
 
+#nullable enable
         internal static bool AccessingAutoPropertyFromConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
         {
-            return AccessingAutoPropertyFromConstructor(propertyAccess.ReceiverOpt, propertyAccess.PropertySymbol, fromMember);
+            if (propertyAccess.UseBackingField == AccessorKind.Unknown)
+            {
+                return false;
```

Comment:
Is there anything we should assert about the property access, when we are asking this question about a property access which may not have had this calculated yet?